Makefile

```diff
@@ -70,6 +70,39 @@ CURL ?= curl
 OC ?= oc
 OCM ?= ocm
 
+### Environment-sourced variables with defaults
+# Can be overriden by setting environment var before running
+# Example:
+#   OCM_ENV=testing make run
+#   export OCM_ENV=testing; make run
+export GOPROXY=https://proxy.golang.org
+export GOPRIVATE=gitlab.cee.redhat.com
+
+ifeq ($(shell uname -s | tr A-Z a-z), darwin)
+	CONTAINER_IMAGE_BUILD_PLATFORM ?= --platform linux/amd64
```

Comment:
Could we double quote the value?

Reply:
updated, note that I had to use echo for this in the image build commands to get rid of the quotes as docker build doesn't accept `"--platform linux/amd64"`

Reply:
Ok then let's unquote it. It seems that Makefile doesn't really interpret single/double quotes and it considers them literally

Reply:
quotes removed 👍